### PR TITLE
Added plugin psucontrol_fritz

### PR DESCRIPTION
**Pull request by not-na**
This adds a sub-plugin to the PSU Control plugin adding support for FRITZ!Box/FRITZ!Dect smart home devices.

### CODE REVIEW

File: _plugins/psucontrol_fritz.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+author: not-na
+license: AGPLv3
+date: 2022-04-27
+tags:
+- power
+- psu
+- control
+- psucontrol
+- psucontrol subplugin
+- tplink
```
Comment (kantlivelong) [inline _plugins/psucontrol_fritz.md:18]: Might want to adjust this :smirk:

**Review by kantlivelong — CHANGES_REQUESTED** [review]
Just two things to adjust.

https://github.com/not-na/OctoPrint-PSUControl-Fritz/blob/4d59a10d7f507ef960e0074a6a7dfdceb19a75db/octoprint_psucontrol_fritz/__init__.py#L146

**Comment by cp2004** [comment]
RE: https://github.com/not-na/OctoPrint-PSUControl-Fritz/blob/4d59a10d7f507ef960e0074a6a7dfdceb19a75db/octoprint_psucontrol_fritz/__init__.py#L57

Will this end up logging usernames and passwords? That is not really what we would want happening and is a security problem. Log files are designed to be shared in case of a problem.

**Comment by kantlivelong** [comment]
> RE

Good catch. Note that I actually do (and thus this fork) log that with [DEBUG](https://github.com/not-na/OctoPrint-PSUControl-Fritz/blob/4d59a10d7f507ef960e0074a6a7dfdceb19a75db/octoprint_psucontrol_fritz/__init__.py#L51) but since this is INFO definitely worth filtering.

File: _plugins/psucontrol_fritz.md (hunk 1 above)
Comment (not-na) [reply _plugins/psucontrol_fritz.md:18]: Oops, sorry. I overlooked that while adjusting the description. I'll fix that right now.

**Comment by not-na** [comment]
I've fixed the logging issue. It's probably an artifact of when I was debugging some issues while creating a session due to an outdated dependency.

**Review by kantlivelong — APPROVED** [review]

**Pull request merged**